main.py

```
@@ -462,16 +462,16 @@ async def delete_like(
+        authorization: Annotated[str, Header(description="로그인 토큰", alias="Authorization")]
 ):
     return Response(status_code=status.HTTP_204_NO_CONTENT)
 
 
 ######### auth ############
 # 회원 로그인
-@app.post("/auth/tokens", response_model=user.LoginUserResponse, status_code=status.HTTP_201_CREATED)
+@app.post("/auth/tokens", response_model=auth.LoginResponse, status_code=status.HTTP_201_CREATED)
 async def auth_token(
-        login_data: Annotated[user.LoginUserRequest, Body()]
+        login_data: Annotated[auth.LoginRequest, Body()]
```

Ruff (0.14.13)

Unused function argument: `login_data`

(ARG001)